src/client/js/otp/util/Itin.js

```diff
@@ -70,7 +70,7 @@ otp.util.Itin = {
     },
 
     isTransit : function(mode) {
-        return mode === "TRANSIT" || mode === "SUBWAY" || mode === "RAIL" || mode === "BUS" || mode === "TRAM" || mode === "GONDOLA" || mode === "TRAINISH" || mode === "BUSISH";
+        return mode === "TRANSIT" || mode === "SUBWAY" || mode === "RAIL" || mode === "BUS" || mode === "TRAM" || mode === "GONDOLA" || mode === "TRAINISH" || mode === "BUSISH" || mode === "AIRPLANE";
```

Comment:
This is exactly as @abyrd commented, `isTransit` is true for all public transit modes, which airplanes are, rather than a car or bike that operate more like walking on the streets and not operated according to a timetable. (Of course we could also add private airplanes, which would be true for something like isNotOnStreetNonTransit)